File: app/src/main/java/com/pivoto/simplesms/MessagingService.java
package com.pivoto.simplesms;

import android.app.Service;
import android.content.Intent;
import android.os.IBinder;
import android.util.Log;

import androidx.annotation.Nullable;

/**
 * Created by pivoto on 17/10/2017.
 */

public class MessagingService extends Service {

    private static final String TAG = "SimpleSMS";

    @Override
    public int onStartCommand(Intent intent, int flags, int startId) {
        Log.d(TAG, "onStartCommand()");
        stopSelf();
        return Service.START_REDELIVER_INTENT;
    }


    @Nullable
    @Override
    public IBinder onBind(Intent intent) {
        return null;
    }
}

File: app/src/main/java/com/pivoto/simplesms/MessagingService.kt
package com.pivoto.simplesms

import android.app.Service
import android.content.Intent
import android.os.IBinder
import android.util.Log

/**
 * Created by pivoto on 17/10/2017.
 */
class MessagingService : Service() {
    override fun onStartCommand(intent: Intent, flags: Int, startId: Int): Int {
        Log.d(TAG, "onStartCommand()")
        stopSelf()
        return START_REDELIVER_INTENT
    }

    override fun onBind(intent: Intent): IBinder? {
        return null
    }

    companion object {
        private const val TAG = "SimpleSMS"
    }
}